services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseTypesTest.java

```diff
@@ -136,34 +137,34 @@ private ContentValues contentValuesTestSeti(int i) {
    }
 
    private void verifyRowTestSet1(TypedRow row) {
-      assertEquals(row.getDataByKey(COL_INTEGER_ID), Long.valueOf(TEST_INT_1));
+      assertEquals(row.getDataByKey(COL_INTEGER_ID), (long) TEST_INT_1);
```

Comment:
row.getDataByKey returns an object. You are removing the conversion to an object so that more casting has to be done. "Long" produces a long oject instead of long.

Reply:
Noted. Thanks.